lib/cli/src/frameworks/angular/1-Button.stories.ts

```diff
@@ -24,7 +24,18 @@ Emoji.args = {
   text: '😀 😎 👍 💯',
 };
 
-Emoji.parameters = { notes: 'My notes on a button with emojis' };
+export const WithCustomBackground = Template.bind({});
+WithCustomBackground.args = {
+  text: 'Defined via addon-backgrounds!',
```

Comment:
How about we just make a story called `Inverse` that:
- Inverts the button background
- Inverts the story background

I think this would fit well into the new template. WDYT?

Reply:
I love it